Update docs about JF_CHECK_ORIGIN (#47)
jellyfish-dev/jellyfish-docs: docs/getting_started/installation.md

Defaults to `5002` for development builds and `8080` for production builds (`mix release` or Docker).
* `JF_SECRET_KEY_BASE` - used to sign/encrypt tokens generated for Peers.
Generated if not provided.

> * `JF_CHECK_ORIGIN` - defines if Jellyfish will check the origin of incoming requests and socket connection. It is possible to specify the origins of requests that will be accepted by Jellyfish.<br/>
> Defaults to `true`.<br/>
> Possible values are `true`, `false` or list of allowed origins space seperated (wildcards are allowed) e.g: `https://example.com //another.com:888 //*.other.com`. 

Reviewer: ```suggestion
* `JF_CHECK_ORIGIN` - definedsadsas if Jellyfish will check the origin of incoming requests and socket connection. <br/>
Defaults to `true`.<br/>
Possible values are `true`, `false` or a space-separated list of allowed origins (wildcards are allowed). <br/>
Example: `JF_CHECK_ORIGIN="https://example.com //another.com:888 //*.other.com"`
```
